File: result/translate/chatgpt/java2python/python/FIND_SUM_UNIQUE_SUB_ARRAY_SUM_GIVEN_ARRAY.py
def f_gold(arr, n):
    res = 0
    m = {}
    
    for i in range(n):
        sum = 0
        for j in range(i, n):
            sum += arr[j]
            if sum in m:
                m[sum] += 1
            else:
                m[sum] = 1
    
    for key, value in m.items():
        if value == 1:
            res += key
    
    return res

File: result/translate/starchat/python2java/java/FIND_SUM_UNIQUE_SUB_ARRAY_SUM_GIVEN_ARRAY.java
public static int f_gold ( int arr [ ], int n ) {
        int res = 0;
        Map < Integer, Integer > m = new HashMap < > ( );
        for ( int i = 0; i < n; i++ ) {
            int Sum = 0;
            for ( int j = i; j < n; j++ ) {
                Sum += arr [ j ];
                m.put ( Sum, m.getOrDefault ( Sum, 0 ) + 1 );
            }
        }
        for ( int x : m.keySet ( ) ) {
            if ( m.get ( x ) == 1 ) {
                res += x;
            }
        }
        return res;
    }